parkings/tests/api/operator/test_parking.py

```diff
@@ -104,7 +104,8 @@ def test_unauthenticated_and_normal_users_cannot_do_anything(api_client, user_ap
     check_method_status_codes(user_api_client, urls, ALL_METHODS, 403, error_code='permission_denied')
 
 
-def test_parking_required_fields(operator_api_client, parking):
+def test_parking_required_fields(operator_api_client, operator, parking):
+    create_enforced_domain_for_user(operator.user)
```

Comment:
Same here. Is this needed? https://github.com/City-of-Helsinki/parkkihubi/pull/146/commits/d2cd37559501da7fbec0130bfe7d4f9ad860b437#r412981973